### PR TITLE
"feat: [es] Create observability-primer.md "

### PR DESCRIPTION
**Pull request by mlunadia**
feat: [es] Create observability-primer.md
Related to https://github.com/open-telemetry/opentelemetry.io/issues/5206

### CODE REVIEW

**Comment by mlunadia** [comment]
@krol3 I see spelling checks are failing for Spanish words, perhaps we need to update the tests to include Spanish or how can we add words to the vocabulary?

File: content/es/docs/concepts/observability-primer.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
+---
+title: Introducción a la Observabilidad
+description: Conceptos básicos de observabilidad.
+weight: 9
+cSpell:ignore: webshop
+default_lang_commit: e58a252c44875b04247b53e2394b4634f5a0a84e
+
+---
+
+## ¿Qué es la Observabilidad?
```
Comment (theletterf) [inline content/es/docs/concepts/observability-primer.md:10]:
```suggestion
## ¿Qué es la observabilidad?
```

File: content/es/docs/concepts/observability-primer.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+comportamiento. Los datos pueden venir en forma de
+[trazas](/docs/concepts/signals/traces/),
+[métricas](/docs/concepts/signals/metrics/) y
+[logs](/docs/concepts/signals/logs/).
+
+**Confiabilidad** responde a la pregunta: "¿Está el servicio haciendo lo que los
+usuarios esperan que haga?" Un sistema podría estar funcionando el 100% del
+tiempo, pero si, cuando un usuario hace clic en "Agregar al carrito" para añadir
+un par de zapatos negros a su carrito, si el sistema no siempre agrega los
+zapatos negros, entonces el sistema podría ser **no** confiable.
```
Comment (theletterf) [inline content/es/docs/concepts/observability-primer.md:44]:
```suggestion
tiempo, pero si, cuando un usuario hace clic en "Agregar al carrito" para añadir
un par de zapatos negros a su carrito, el sistema no siempre agrega los
zapatos negros, entonces el sistema podría ser **no** confiable.
```

File: content/es/docs/concepts/observability-primer.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+**Métricas** son agregaciones durante un período de tiempo de datos numéricos
+sobre tu infraestructura o aplicación. Ejemplos incluyen: tasa de error del
```
Comment (theletterf) [inline content/es/docs/concepts/observability-primer.md:47]:
```suggestion
sobre tu infraestructura o aplicación. Por ejemplo: tasa de error del
```

File: content/es/docs/concepts/observability-primer.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+### Spans
+
+Un **span** representa una unidad de trabajo u operación. Los spans rastrean
+operaciones específicas que realiza una solicitud, mostrando qué sucedió durante
+el tiempo en que se ejecutó esa operación.
+
+Un span contiene nombre, datos relacionados con el tiempo,
+[mensajes de log estructurados](/docs/concepts/signals/traces/#span-events) y
+[otros metadatos (es decir, Atributos)](/docs/concepts/signals/traces/#attributes)
```
Comment (theletterf) [inline content/es/docs/concepts/observability-primer.md:105]:
```suggestion
[otros metadatos (es decir, atributos)](/docs/concepts/signals/traces/#attributes)
```

**Review by theletterf — CHANGES_REQUESTED** [review]
Excelente trabajo, @mlunadia, tan solo una sugerencias de estilo (que estoy consolidando en un documento).

Sobre los errores de spelling, el diccionario no es del todo completo. Sugiero añadir las excepciones en el frontmatter usandp `cSpell:ignore:` o bien a https://github.com/open-telemetry/opentelemetry.io/blob/main/.cspell/es-palabras.txt

**Comment by mlunadia** [comment]
Gracias @theletterf por las sugerencias!

**Comment by theletterf** [comment]
@mlunadia The spelling errors are still there, please check them out.

**Comment by mlunadia** [comment]
@theletterf ah, got it. added them to the txt file, pending [PR](https://github.com/open-telemetry/opentelemetry.io/pull/5320) approval.

**Review by theletterf — APPROVED** [review]

**Pull request merged**